### PR TITLE
docs: Several docs-related tweaks and updates

### PR DESCRIPTION
**Pull request by ldoktor**
Several docs update related to docs upkeep...

### CODE REVIEW

**Comment by ldoktor** [comment]
[Sprint Maintainance Tasks (#46)](https://trello.com/c/3Cu4GL13/892-sprint-maintainance-tasks-46)

File: docs/source/WritingTests.rst [hunk 1]
```diff
@@ -747,14 +747,6 @@ namely the ``--output-check-record`` argument with values ``stdout``,
 to the files ``stdout.expected`` and ``stderr.expected`` at the test's
 data directory (which is different from the job/test results directory).
 
-Avocado Tests run on a separate process
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: OK, agreed.

File: docs/source/WritingTests.rst [hunk 2]
```diff
@@ -62,9 +62,11 @@ Saving test generated (custom) data
 
 Each test instance provides a so called ``whiteboard``. It can be accessed
 through ``self.whiteboard``. This whiteboard is simply a string that will be
-automatically saved to test results (as long as the output format supports it).
-If you choose to save binary data to the whiteboard, it's your responsibility to
-encode it first (base64 is the obvious choice).
+automatically saved to test results after the test finishes (it's not synced
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: OK.

File: docs/source/Loaders.rst [hunk 3]
```diff
@@ -52,6 +52,26 @@ as some of them result in error)::
...
+
+This will run 3 tests, the first one is a normal test defined by
+``existing_file.py`` (most probably an instrumented test). Then
+we have ``/bin/echo`` which is going to be executed via
+``/bin/sh -c '/bin/echo something'``. The last one would be
+``nonexisting-file`` which would execute ``/bin/sh -c nonexisting-file``
+which most probably fails.
+
+Note that you are responsible for quotating the test-id (see the
+``"'/bin/echo something'"`` example).
```
Comment (clebergnu) [inline docs/source/Loaders.rst]: OK.

File: docs/source/WritingTests.rst [hunk 4]
```diff
@@ -873,7 +876,7 @@ Another way to skip tests is by using the Avocado skip decorators:
   ``False``
 
 Those decorators can be used with both ``setUp()`` method and/or and in the
-test methods. The test below::
+``test*()`` methods. The test below::
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: OK.

**Pull request merged**